src/renderer/components/+preferences/preferences.tsx

```diff
@@ -42,21 +43,32 @@ import { KubeconfigSyncs } from "./kubeconfig-syncs";
 import { SettingLayout } from "../layout/setting-layout";
 import { Checkbox } from "../checkbox";
 import { sentryDsn } from "../../../common/vars";
+import { isNumber } from "../input/input_validators";
 
 enum Pages {
   Application = "application",
   Proxy = "proxy",
   Kubernetes = "kubernetes",
   Telemetry = "telemetry",
+  Editor = "editor",
   Extensions = "extensions",
   Other = "other"
 }
 
+enum EditorLineNumbersStyles {
+  default = "Default",
+  on = "On",
+  off = "Off",
+  relative = "Relative",
+  interval = "Interval"
+}
```

Comment:
This enum does not correspond to the type `monaco.editor.LineNumbersType` (at least from the code in this PR). Namely, you have set the default value to be `"on"` but that value is not a valid variant of this enum.

Reply:
I set enum's value as a parameter in setLineNumbers(),  it works without issues:
![image](https://user-images.githubusercontent.com/53330707/127113338-3c1429dc-55a5-45cc-a3f8-bc99e04067cd.png)

Also, LineNumbersType allows to use "on":
  export type LineNumbersType = 'on' | 'off' | 'relative' | 'interval' | ((lineNumber: number) => string);

Reply:
That is because select will just default to "string" in that case.

Reply:
You prefer literals? Also I don't think we should support the line number function to begin with.